boards/wemos/esp32s2_lolin_mini/doc/index.rst

```diff
@@ -74,20 +74,44 @@ application.
    :board: esp32s2_lolin_mini
    :goals: flash
 
-Open a serial port using e.g. screen
+Perform the needed connections (see: Helper hardware connections section)
+to see :ref:`hello_world` output on the screen and open a serial port using e.g. screen:
 
 .. code-block:: shell
 
    screen /dev/ttyUSB0 115200
 
+Or using espressif monitor
+
+.. code-block:: shell
+
+   west espressif monitor
+
+
 After the board has been manually reset and booted, you should see the following
-message in the monitor:
+message in the monitor (see: Hardware Connections section)
 
 .. code-block:: console
 
    ***** Booting Zephyr OS vx.x.x-xxx-gxxxxxxxxxxxx *****
    Hello World! esp32s2_lolin_mini
 
+Helper Hardware connections
+---------------------------
+
+.. warning::
+
+   To this date esp32s2 SoC does not possess USB CDC-ACM support in zephyr.
```

Comment:
@raffarost just curious how easy it would be to enable CDC-ACM on S2? It looks like it was reasonably easy for C6 so might be the same for the other ESP32 series?

Reply:
Hi @kartben 
I believe it might be possible to enable it like we did in C6, indeed.

Reply:
The ESP32-S2 has much lower memory compared to other ESP32xx, therefore it can be bit tricky to enable all nice features at once :)

Reply:
It won't be as simple. It doesn't feature the hardware support like newer Espressif Chips (starts with ESP32-C3). From [this link](https://docs.espressif.com/projects/esp-idf/en/v5.3/esp32c6/api-guides/usb-serial-jtag-console.html):

> The USB Serial/JTAG Controller is a fixed-function USB device that is implemented entirely in hardware, meaning that it cannot be reconfigured to perform any function other than a serial port and JTAG debugging functionality. This is in contrast to the USB OTG controllers in some ESP chips that can be configured to perform the function of multiple types of USB devices.

It would have to be a software based implementation, using the USB-OTG. But for that, we need to first support the USB. Then, I imagine it will be normal Zephyr USB-CDC-ACM.

Also, for the sake of further education to those reading this, ESP32-S3, for example, features BOTH the USB-OTG and the USB-CDC. ESP32-S2 only features the USB-OTG and ESP32-C3 only features the USB-CDC.

Reply:
Anything I can help on? Eager to help :)